packages/parcel-transformer/src/index.ts

```diff
@@ -124,6 +124,12 @@ export default new Transformer<ParcelTransformerOpts>({
   },
 
   async transform({ asset, config, options }) {
+    if (config.extract && config.classHashPrefix) {
```

Comment:
note that `config.extract` is currently ignored in non-production environments when using Parcel

Reply:
Got it, this kind of might be the wrong thing in that case if there's no different configs…I'm not sure exactly, might wait to see the PR from @guilhermehto to see if this would blow up or not.